File: benchmark/results/python-multilang/run-004/a-mcp/output/catalog/aggregates/product.py
from datetime import datetime

"""Product aggregate root for the catalog domain."""
class Product:
    def __init__(self, id: str, created_at: datetime, updated_at: datetime, name: str, description: str):
        self.id = id
        self.created_at = created_at
        self.updated_at = updated_at
        self.name = name
        self.description = description

File: benchmark/results/java-multilang/run-003/b-baseline/output/com/metaengine/demo/catalog/aggregates/Product.java
package com.metaengine.demo.catalog.aggregates;

import java.time.Instant;

/** Product aggregate root for the catalog domain. */
public class Product {
    private final String id;
    private final Instant createdAt;
    private final Instant updatedAt;
    private final String name;
    private final String description;

    public Product(String id, Instant createdAt, Instant updatedAt, String name, String description) {
        this.id = id;
        this.createdAt = createdAt;
        this.updatedAt = updatedAt;
        this.name = name;
        this.description = description;
    }

    public String getId() { return id; }
    public Instant getCreatedAt() { return createdAt; }
    public Instant getUpdatedAt() { return updatedAt; }
    public String getName() { return name; }
    public String getDescription() { return description; }
}
